Cargo.toml

```diff
@@ -1,27 +1,31 @@
-[package]
```

Comment:
do you have any idea why this is showing all these changes? Did the encoding change or something??

edit: looked into this, and it's a different line ending?

Reply:
Weird. My git settings should be enforcing LF. Sorry about that